### PR TITLE
improvement(cdc_repl_jenkins): Switch cdc_repl pipeline to use longev…

### PR DESCRIPTION
**Pull request by aleksbykov**
…ity pipeline

Switch cdc repl pipeline to longevity pipeline for future support and compatibility

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: cdc_replication_test.py [hunk 1]
```diff
@@ -197,6 +212,23 @@ def test_replication(self) -> None:
             self.log.info('Consistency check successful.')
         else:
             self.collect_data_for_analysis(
-                    migrate_log_path, replicator_log_path,
-                    master_node, replica_node)
+                migrate_log_path, replicator_log_path,
+                master_node, replica_node)
             self.fail('Consistency check failed.')
+
+    def get_email_data(self):
```
Comment (kbr-) [inline cdc_replication_test.py:219]: Where is this function used?

File: test-cases/cdc/cdc-15m-replication.yaml [hunk 2]
```diff
@@ -10,6 +10,7 @@ instance_type_db: 'i3.large'
 
 n_test_oracle_db_nodes: 1
 instance_type_db_oracle: 'i3.large'
+oracle_scylla_version: 'master:latest'
```
Comment (kbr-) [inline test-cases/cdc/cdc-15m-replication.yaml:13]: I would like to be able to run this pipeline on different Scylla versions too.

File: cdc_replication_test.py (hunk 1 above)
Comment (aleksbykov) [reply cdc_replication_test.py:215]: @kbr- it is required to build email report

**Comment by aleksbykov** [comment]
@kbr- Added new pipeline. It use most of longevity functionality
also added new email template

**Comment by roydahan** [comment]
@bentsi please review

**Comment by kbr-** [comment]
If I understand correctly, this new pipeline will also destroy the cluster even if the test fails. It will only keep in on critical (framework) errors, as described in https://github.com/scylladb/scylla-cluster-tests/issues/1825.

Would it be possible to configure the pipeline as follows?
1. if the test succeeds, destroy the cluster,
2. if the test fails, leave the cluster running for 2 days, so that someone can investigate.
After 2 days, destroy it, unless someone explicitly says that they want to keep it longer.

---

Technical comment: why set environment variables from the bash script? Jenkins has the `environment` pipeline block, which is AFAIK the idiomatic way to set env variables.

**Comment by roydahan** [comment]
It will be possible once our PR will be merged to Scylla-devops.
(PR to set the keep-alive till certain date).
For the first part, we have the keep-on-failure.

**Pull request merged**